### PR TITLE
feat(metrics-summaries): Ingest new segment-related columns

### PR DESCRIPTION
**Pull request by phacops**
Once https://github.com/getsentry/snuba/pull/5453 is merged and clusters are migrated, we can start ingesting new fields.

### CODE REVIEW

File: snuba/datasets/configuration/spans/storages/metrics_summaries.yaml [hunk 1]
```diff
@@ -67,9 +71,6 @@ query_processors:
   - processor: UUIDColumnProcessor
     args:
       columns: [trace_id]
-  - processor: HexIntColumnProcessor
```
Comment (untitaker) [inline snuba/datasets/configuration/spans/storages/metrics_summaries.yaml:70]: was this change intentional? I'm not really sure, I think this changes the type visible on the query layer, so any query using span_id would've to be fixed.

Comment (phacops) [reply snuba/datasets/configuration/spans/storages/metrics_summaries.yaml:70]: This is in the storage configuration, so should only matter at write time. From what I understand, this will transform a string to a integer when the string is a hexadecimal value. In this case, we're already transforming the data in Rust so there's not need for that processor.

Comment (phacops) [reply snuba/datasets/configuration/spans/storages/metrics_summaries.yaml:70]: I thought this was only applicable at write time (and it already arrives as an integer), but it is also in charge of transforming the value back to an hexadecimal string.

I added the relevant columns instead of removing it.

**Review by evanh — APPROVED** [review]

**Review by nikhars — APPROVED** [review]
